Makefile

```diff
@@ -2,13 +2,15 @@ include ./config.mk
 
 TAGNAME ?= default
 
+#========================= Run tests with Docker locally =========================#
```

Comment:
if you want u can push no changes to make file, since on the next PR it will overwrite almost all .

Reply:
If your PR gets merged before mine, I can rebase, its not biggie